### DISCUSSION TITLE
Proxmox - Netbox

### DISCUSSION THREAD

**Discussion by Peachy74** [netbox-community/netbox#11799] [General]
Hello everyone
I'm trying to use the netbox-proxbox plugin to extract the information from the Proxmox.
A connection to the Proxmox cluster seems to work.
However, I still get an error message:

`<class 'pynetbox.core.query.RequestError'>

The request failed with code 400 Bad Request but more specific details were not returned in json. Check the NetBox Logs or investigate this exception's error attribute.

Python version: 3.10.7
NetBox version: 3.3.9`

someone an idea how I can grope my way forward
regards
Alex

**Comment by candlerb** [comment]
If you mean https://github.com/netdevopsbr/netbox-proxbox then I'd suggest you raise the issue there.

Check first whether this plugin has been tested with Netbox 3.3 (it only says ">=3.2.0").   Netbox doesn't have any backwards compatibility, so something which worked with v3.2 could well not work with v3.3 without changes.

Also you may want to provide them with a full backtrace: you can do this by setting ADMINS and EMAIL in configuration.py, then the exception details should be E-mailed to you.
